README.md

```diff
@@ -2,15 +2,15 @@
 
 ## Introduction
 
-The Deprecation Toolkit is a gem to help you get rid of deprecations in your codebase.
-Having deprecations in your application is usually a sign that something will break whenever a third dependency will get updated. The sooner the better to fix them!
-Fixing all deprecations at once might be though depending on how big your app is and how much deprecations needs to be fixed. You might have to progressively resolve them while making sure your team doesn't add new one ➰. This is where this gem comes handy!
+Deprecation Toolkit is a gem that helps you get rid of deprecations in your codebase.
```

Comment:
Might be because english is not my first language but it sounds weird to not use a pronoun "**The** Deprecation Toolkit"